thriftserver/thriftserver/base/thriftserver-db-dc.yaml

```diff
@@ -0,0 +1,101 @@
+---
+apiVersion: apps.openshift.io/v1
+kind: DeploymentConfig
+metadata:
+  name: thriftserver-db
+  annotations:
+    template.alpha.openshift.io/wait-for-ready: "true"
+spec:
+  strategy:
+    type: Recreate
+  triggers:
+    - type: ImageChange
+      imageChangeParams:
+        automatic: true
+        containerNames:
+          - postgresql
```

Comment:
Noted above, but just to be sure - can we use images from RH registry?

Reply:
Yes we can! :+1: